src/nacl/signing.py

```diff
@@ -17,7 +17,9 @@
 import six
 
 from nacl import encoding
+
 import nacl.bindings
+from nacl.public import PrivateKey, PublicKey
```

Comment:
This one bit makes me nervous: we don't have an `__all__`, so this will get re-exported. That means someone could accidentally do `from nacl.signing import PublicKey` and think they're getting an Ed25519 verifying key, but they're really getting an out-of-place Curve25519 public key.

Could we maybe add an `as _Curve25519_PrivateKey` (etc) in there?